reporters/cloud-run-scripts/CloudResponce.js

```diff
@@ -0,0 +1,27 @@
+const express = require("express");
+const app = express();
+const { exec } = require("child_process");
+
+app.get("/", (req, res) => {
+  console.log("Daily reporter received a request");
+
+  let reporterScript = exec("/bin/bash ../reporters/cloud-run-scripts/SendSlackReport.sh", (error, stdout, stderr) => {
+    console.log(stdout);
+    console.log(stderr);
+
+    if (error !== null) {
```

Comment:
So will this `error` be thrown for example if the `jq` command in `bash` throws?

Reply:
Yup. if the process ends with anything other than `exit 0` it will have an error. In my testing, malformed queries on the `curl` command to the slack API are correctly caught in this.